targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L443xC/TOOLCHAIN_ARM/stm32l443xc.sct

```diff
@@ -102,7 +102,7 @@ LR_IROM1  MBED_APP_START  MBED_APP_SIZE  {    ; load region size_region
    .ANY (+RW +ZI)
   }
 
-  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE+MBED_IRAM2_START-AlignExpr(ImageLimit(RW_IRAM1), 16))  {
+  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE-(MBED_RAM_START-AlignExpr(ImageLimit(RW_IRAM1), 16)))  {
```

Comment:
I think it's the other way round?
```suggestion
  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE-(AlignExpr(ImageLimit(RW_IRAM1), 16)-MBED_RAM_START))  {
```
because `ImageLimit(RW_IRAM1)` is RW_IRAM1's end address, higher than the start address